### DISCUSSION TITLE
Non-loop cue point

### DISCUSSION THREAD

**Discussion by maximeferrieu** [Zeugma440/atldotnet#105] [Q&A] [answered]
Is there a way to detect one-shot embed markers (cue-points) and not only loop regions?
They don't seem to appear in the `Track.AdditionalFields`

Thanks for your anwser!

**Comment by Zeugma440** [comment, marked as answer]
Heya !

I need a little more context to even start answering that one : which audio format are we talking about ? 😁 

The notion of loop is present on various formats, I can't just guess by reading what you said...

**Reply by maximeferrieu** [reply]
Hey, thanks for your answer :)
It's a .wav that has 2 loop regions and 6 "one-shot" cue points.
The loops regions are found properly when accessing `Track.AdditionalFields`

```
foreach(string key in track.AdditionalFields.Keys)
{
     Debug.Log(key.ToString() + "= " + track.AdditionalFields[key]);
}
```


gives me `sample.NumSampleLoops= 2`, so this is correct, but it does'nt seem to display the "one shot" cue point, which I am sure are read at some point because the CuePointId of the SampleLoop is displayed `sample.SampleLoop[0].CuePointId= 2`

On top of that, is there any way to acces the names of those markers and loop regions?

Thank you very much for your time!

**Reply by Zeugma440** [reply]
Oh sure... loop regions are supported because someone requested them. It shouldn't be too hard to add cue points support either. 

I'll see what I can do 😉

**Reply by Zeugma440** [reply]
Follow-up in https://github.com/Zeugma440/atldotnet/issues/106

**Reply by maximeferrieu** [reply]
Well that's awesome, thank you very much for opening this!
